RELEASING.md

```diff
@@ -58,7 +60,18 @@ release is announced.
 
 ### Performing a "patch" release
 Patch releases shouldn't include any new functionality, just bug fixes (expect during pre-release testing).
-1. Add the new version to `_data/versions.yaml` (Remember to add the date in the description)
+1. Add the new version to `_data/versions.yaml`:
+   - Add the date in the description.
+   - Make sure new release candidate versions are added to the top of the new release section (otherwise
+     yaml will incorrectly identify an older release candidate):
+    ```
```

Comment:
this triple-code-tick (and its closing counterpart) needs 1 more space in front of it to render properly.

Reply:
fixed in latest push